src/Models/SortByDragAndDrop.php

```diff
@@ -27,14 +27,6 @@
  */
 trait SortByDragAndDrop
 {
-    /**
-     * @return string[]
-     */
-    private static function updateSortIndexRule(): array
-    {
-        return ['string', 'regex:/^(?:after|before):' . RestController::UUID_V4_PATTERN . '$/'];
-    }
-
```

Comment:
Still not sure if I want really remove this method, because would be a breaking-change for the library code ..